### ISSUE TITLE
Add feature to extract report from the dashboard

### ISSUE THREAD

**Issue by waseemahammed** [jenkinsci/devops-portal-plugin#9]
### What feature do you want to see added?

It would be great if we could extract the devops metrics from the dashboard for further analysis. May be we can create a dashboard to to track DORA metrics which we capture from RUN and DEPLOYMENT metrics.

```[tasklist]
### Tasks
```

**Comment by rbello** [comment]
DORA metrics :

- Deployment Frequency —How often an organization successfully releases to production
- Lead Time for Changes —The amount of time it takes a commit to get into production
- Change Failure Rate —The percentage of deployments causing a failure in production
- Time to Restore Service —How long it takes an organization to recover from a failure in production

**Comment by waseemahammed** [comment]
Yes. Can we add a feature to export these metrics to csv any other format?
